docs/Home.md

```diff
@@ -0,0 +1,31 @@
+# Fee Oracle technical documentation
+
+Fee Oracle service
+facilitates [Sygmas Dynamic fee strategy](https://github.com/sygmaprotocol/sygma-relayer/blob/main/docs/general/Fees.md),
+allowing users to request a **signed fee estimate** through its API.
+
+To request a fee estimate, users must provide the following parameters:
+
+- **fromDomainID** - the identifier of the source domain (chain)
+- **toDomainID** - the identifier of the destination domain (chain)
+- **resourceID** - the identifier of the cross-chain resource
+
+When requesting a fee estimate for a generic message, depending on the destination domain (network), users must also provide the following query parameter:
+
+**EVM**
+
+- **msgGasLimit** - the maximum amount of gas units that can be used on execution
```

Comment:
This is a hardcoded value? Or it is taken from the config? IF it is then by what param?